### PR TITLE
[templates] update hello world templates to work with latest cli

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

File: templates/00-hello-world/src/local_test_environment.js [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+const nearlib = require('nearlib');
+
+
+class LocalTestEnvironment extends NodeEnvironment {
+    constructor(config) {
+        super(config);
+    }
+
+    async setup() {
+        this.global.ArrayBuffer = ArrayBuffer;
```
Comment (vgrichina) [inline templates/00-hello-world/src/local_test_environment.js]: hmm, why is it needed to copy all the already global stuff to `this.global`?

File: templates/00-hello-world/src/local_test_environment.js [hunk 2]
```diff
@@ -0,0 +1,82 @@
+const NodeEnvironment = require('jest-environment-node');
+const dev = require('nearlib/dev');
+const fs = require('fs');
+const nearlib = require('nearlib');
+
+
+class LocalTestEnvironment extends NodeEnvironment {
```
Comment (vgrichina) [inline templates/00-hello-world/src/local_test_environment.js]: why not ship this as part of nearlib package which isn't compiled for browser?

File: templates/00-hello-world/src/local_test_environment.js (hunk 1 above)
Comment (janedegtiareva) [reply templates/00-hello-world/src/local_test_environment.js]: There is some obscure buggy behaviour of node test environment and versions of dependencies it uses. Something in buffers starts breaking. Sadly this is the recommended solution.

File: templates/00-hello-world/src/local_test_environment.js (hunk 2 above)
Comment (janedegtiareva) [reply templates/00-hello-world/src/local_test_environment.js]: This is now a part of shell new project. I can use this PR to bring this in

File: templates/00-hello-world/src/local_test_environment.js (hunk 1 above)
Comment (vgrichina) [reply templates/00-hello-world/src/local_test_environment.js]: >  Sadly this is the recommended solution.

May you insert link to this? Would help whoever is reading code and has same question.

**Comment by vgrichina** [comment]
@janedegtiareva do you plan to merge this?

**Pull request closed without merging**